src/scripting/game_lua_kernel.cpp

```diff
@@ -2750,6 +2750,44 @@ int game_lua_kernel::intf_set_floating_label(lua_State* L, bool spawn)
 	return 1;
 }
 
+namespace
+{
+	const unit_map& get_unit_map()
+	{
+		// Used if we're in the game, including during the construction of the display_context
+		if(resources::gameboard) {
+			return resources::gameboard->units();
+		}
+
+		// If we get here, we're in the scenario editor
+		assert(display::get_singleton());
+		return display::get_singleton()->context().units();
+	}
+	void reset_affect_adjacent(const unit& u_)
```

Comment:
`u_` is a confusing name for a function paramater.

Reply:
changed to 'unit'